src/frontend/staff-portal/src/app/components/staff-workbench/dispute-decision-inbox/dispute-decision-inbox.component.ts

```diff
@@ -64,29 +72,118 @@ export class DisputeDecisionInboxComponent implements OnInit, AfterViewInit {
     });
   }
 
+  onApplyFilter(filterName: string, value: string) {
+    const filterValue = value;
+    this.dataFilters[filterName] = filterValue;
+    this.dataSource.filter = JSON.stringify(this.dataFilters);
+    this.tableHeight = this.calcTableHeight(352);
+  }
+  
+  resetSearchFilters() {
+    // Will update search filters in UI
+    this.dataFilters = {
+      "dateFrom": "",
+      "dateTo": "",
+      "ticketNumber": "",
+      "occamDisputantName": "",
+      "courthouseLocation": "",
+      "team": ""
+    };
+
+    // Will re-execute the filter function, but will block UI rendering
+    // Put this call in a Timeout to keep UI responsive.
+    setTimeout(() => {
+      //this.data = this.sampleData();
+      this.dataSource.filter = "{}";
+      this.tableHeight = this.calcTableHeight(352);
+    }, 100);
+  }
+  
+  searchFilter = function (record: JJDispute, filter: string) {
+    let searchTerms = JSON.parse(filter);
+    return Object.entries(searchTerms).every(([field, value]: [string, string]) => {
+      if ("dateFrom" === field) {
+        return !DateUtil.isValid(value) || DateUtil.isDateOnOrAfter(record.jjDecisionDate, value);
+      }
+      else if ("dateTo" === field) {
+        return !DateUtil.isValid(value) || DateUtil.isDateOnOrBefore(record.jjDecisionDate, value);
```

Comment:
Once this PR is in main, I plan to reuse the new DateUtil on the other tabs as well.